src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,10 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
-public interface MachineProducer {
-    /**
-     * In your implementations - create 2-3 models, add them to the list and return
-     * @return - the list of models
-     */
-    List<Object> get();
+public interface MachineProducer<E> {
+    int LIST_SIZE = 3;
```

Comment:
don`t forget about access modifiers is package private is the best option here? Also rename this to represent its sence like default number of machines

Reply:
It is not package protected, it is public static final and I cannot change its access parameters. Did you mean that I need to create a private static final variable separately in each class? Although they will be a duplication in this case and do not fit the  DRY principle.